projects/migrations/0013_add_tag.py

```diff
@@ -141,6 +141,8 @@ def create_tags(apps, _):
 
     tag_model.objects.bulk_create(tags)
 
+    if type_model.objects.count() == 0:
```

Comment:
Why is the migration being updated? This already has been run on prod right?

Reply:
Yeah, it won't run again. This is just to catch if people want to run on their local deployments. The migration fails if you have an empty types/projects table. This just short circuits it.